pwiz_tools/Skyline/Model/DiaUmpireDdaConverter.cs

```diff
@@ -143,16 +143,18 @@ public override bool Run(IProgressMonitor progressMonitor, IProgressStatus statu
                     if (File.Exists(outputFilepath))
                         FileEx.SafeDelete(outputFilepath);
 
-                    string tmpFilepath = Path.Combine(Path.GetTempPath(), PathEx.GetRandomFileName() + MsConvertOutputExtension); // N.B. FileEx.GetRandomFileName adds unusual characters in test mode
-                    string tmpParams = Path.Combine(Path.GetDirectoryName(outputFilepath) ?? string.Empty, @$"diaumpire-{DateTime.Now.ToString(@"yyyyMMddhhmm")}.params");
+                    // DiaUmpire doesn't handle unicode paths

```

Comment:
It should, actually. It's all pwiz C++ code. Did you actually run into problems with it or it just got caught up in a "uses ProcessRunner" search-and-replace?